src/web/components/ParticipantRequests/ApprovedParticipantItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { ParticipantRequestDTO } from '../../../api/routers/participantsRouter';
+
+import './ParticipantRequestItem.scss';
+
+type ApprovedParticipantProps = {
+  participant: ParticipantRequestDTO;
+};
+
+export function ApprovedParticipantItem({ participant }: ApprovedParticipantProps) {
+  function getParticipantTypes(
+    currentParticipantTypes?: ApprovedParticipantProps['participant']['types']
+  ) {
+    if (!currentParticipantTypes) return null;
+    return currentParticipantTypes.map((pt) => (
+      <div className='participant-request-type-label' key={pt.typeName}>
+        {pt.typeName}
+      </div>
+    ));
+  }
+
+  return (
+    <tr className='participant-request-item'>
```

Comment:
is this className right?

Reply:
I changed this, but I intend to combine the components when I come back to this.